app/services/current-user.ts

```diff
@@ -117,6 +117,7 @@ export default class CurrentUserService extends Service {
     async checkShowTosConsentBanner(this: CurrentUserService) {
         const user = await this.user;
         if (user && !user.acceptedTermsOfService) {
+            user.set('acceptedTermsOfService', null);
```

Comment:
This totally, works, so well done. However, if you hit continue, close the box, and transition, when the box comes back, the validation error is still in place. Can you fix that as well while you're here? If not, this is still better, but that's my only other quibble.

Reply:
Actually, one other quibble, which are the failing tests.